images/basic/fuse/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN set -x \
+    && apt-get update \
+    && apt-get install -y \
+    wget \
+    git \
+    pkg-config \
+    zip \
+    g++ \
+    zlib1g-dev \
+    unzip \
+    python3 \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update && apt-get install -y software-properties-common
+
+# Get everything needed for a FUSE Server.
+RUN add-apt-repository ppa:ubuntu-toolchain-r/test
+RUN apt-get install -y build-essential fuse3 libfuse3-3 libfuse3-dev strace
+RUN apt-get install -y procps vim libstdc++6
+
+WORKDIR /fus
+
+RUN mkdir -pv mountpoint
+RUN git clone https://github.com/libfuse/libfuse
+RUN gcc -Wall ./libfuse/example/hello.c `pkg-config fuse3 --cflags --libs` -o server-bin
```

Comment:
This should be `passthrough.c`. I think what the image does is to build a `passthrough` and enable fuse library. Do you think the following `Dockerfile` work? Why do we need software-properties-common and add toolchain repo?
```
FROM ubuntu:20.04

ENV DEBIAN_FRONTEND=noninteractive

RUN apt-get update
RUN apt-get install -y build-essential git pkg-config fuse3 libfuse3-3 libfuse3-dev strace

WORKDIR /fus

RUN git clone https://github.com/libfuse/libfuse
RUN mkdir -pv mountpoint

RUN gcc -Wall ./libfuse/example/passthrough.c `pkg-config fuse3 --cflags --libs` -o server-bin

CMD ["bash"]
```